pandas/tools/util.py

```diff
@@ -61,6 +61,42 @@ def to_numeric(arg, errors='raise'):
         - If 'raise', then invalid parsing will raise an exception
         - If 'coerce', then invalid parsing will be set as NaN
         - If 'ignore', then invalid parsing will return the input
+    downcast : {'signed', 'unsigned'}, str, numpy.dtype, or
+               Python dtype, default None
+        If possible, downcast the resulting numerical data to the
+        specified numerical dtype in 'downcast'. If the size
+        of the data's dtype is smaller or equal to that of 'downcast',
+        then this parameter will be ignored.
+
+        If 'downcast' refers to a more generic dtype (e.g. int),
+        downcast the resulting numerical data to the smallest
+        dtype possible under that more generic dtype specification
+        that is of smaller size than the current numerical
+        data dtype according to the following rules:
+
+        1) int
+
+           Smallest signed int dtype. To select this option,
+           pass in 'int', 'signed', np.int, or int.
+
+        2) uint
+
+           Smallest unsigned int dtype. To select this option,
+           pass in 'uint' or 'unsigned'.
+
+        3) float
+
+           Smallest float dtype. To select this option,
+           pass in 'float', np.float, or float.
+
+        If the specification cannot be followed (e.g. trying to
```

Comment:
so I would do this only for (this is a uint request with < 0 numbers)

- ignore -> don't do anything 
- coerce -> transform as you suggest
- raise -> raise (I think you do this now), are testing?